src/dispatcher/dispatch-actions/threads.ts

```diff
@@ -14,46 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { MatrixEvent } from "matrix-js-sdk/src/models/event";
-
 import RightPanelStore from "../../stores/right-panel/RightPanelStore";
 import { RightPanelPhases } from "../../stores/right-panel/RightPanelStorePhases";
-import dis from "../dispatcher";
-import { Action } from "../actions";
-import { TimelineRenderingType } from "../../contexts/RoomContext";
-
-export const showThread = (props: {
```

Comment:
nitpick: Why not keep this here as a utility and call it from the action handler in `MatrixChat`?

Reply:
Also, whether here or in `MatrixChat`, it would nice to have some test coverage for the logic here.

Reply:
> nitpick: Why not keep this here as a utility and call it from the action handler in `MatrixChat`?

I think it makes sense for consistency - it will avoid some code using the dispatcher and some code using the utility itself